model2vec/model.py

```diff
@@ -186,24 +190,40 @@ def encode_as_sequence(
             If this is None, no truncation is done.
         :param batch_size: The batch size to use.
         :param show_progress_bar: Whether to show the progress bar.
+        :param use_multiprocessing: Whether to use multiprocessing.
```

Comment:
if you want you can add additional information about constraints here.

Reply:
Good point, will update